mittab/apps/tab/views.py

```diff
@@ -248,4 +272,47 @@ def view_scratches(request):
                              {'item_type':'team',
                               'title': "Viewing All Scratches for Teams",
                               'item_list':c_scratches}, context_instance=RequestContext(request))
+
+def upload_data(request):
+    if request.method == 'POST':
+      form = UploadDataForm(request.POST, request.FILES)
+      if form.is_valid():
+        judgeErrors = scratchErrors = teamErrors = []
+        importName = ''
+        results = ''
+        #TODO: Could probably turn these checks into a loop
+        if 'teamFile' in request.FILES:
```

Comment:
I was initially concerned about users having inconsistent capitalization of school names, especially when abbreviated (ie GTown vs Gtown), but I realized as I was responding Django allows you to check for this with name__iexact. Schools weren't added automatically before, but I'll update it.